extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -31,7 +31,7 @@
  */
 .amp-lightboxed {
   position: fixed !important;
-  z-index: 2147483645 !important;
+  z-index: 2147483644 !important;
```

Comment:
as we do need to fix things for changing the z-index, let's fix them at same time?